README.md

```diff
@@ -543,6 +543,59 @@ Adjust `tsconfig.json` jsx factory function option.
 
 ```
 
+#### Use React with `<Script />`
+
+If you export a manifest file in `dist/.vite/manifest.json`, you can easily write some codes using `<Script />`.
+
+```tsx
+// app/routes/_renderer.tsx
+import { reactRenderer } from '@hono/react-renderer'
```

Comment:
Please add the import statement for `Script`:

```ts
import { Script } from 'honox/server'
```

Reply:
OK. Added it.